manifests/master/base.pp

```diff
@@ -56,6 +59,14 @@
     notify => Service[$::pupmod::master::service]
   }
 
+  $auth_conf = '/etc/puppetlabs/puppetserver/conf.d/auth.conf'
+
+  puppet_authorization { $auth_conf:
+    version => 1,
+  }
+
+  Service[$::pupmod::master::service] ~> Exec <| title == 'simp_generate_types' |>
```

Comment:
Curious, why not just flip the dependency line in `master.pp`?

Reply:
Because the `simp_generate_types` exec can be disabled.  We don't want to fail to start the service just because the exec is missing.

Reply:
@pillarsdotnet Yes, it does. Basically, adding a dependency on a class (or notifying one) causes a direct relationship on all resources contained within the class.

Reply:
I'll press the "I believe" button...